src/applications/disability-benefits/all-claims/tests/components/ToxicExposureChoicePage.unit.spec.jsx

```diff
@@ -0,0 +1,867 @@
+    it('displays the page title and checkboxes', () => {
+      const data = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        toxicExposure: { conditions: {} },
+      };
+
+      const { container } = renderPage({ data });
+
+      expect(container.textContent).to.contain('Toxic exposure');
+      expect(document.querySelector('va-checkbox-group')).to.exist;
+      expect(container.textContent).to.contain(
+        'Learn more about toxic exposure',
+      );
+    });
+
+    it('displays forward and back buttons on the form page', () => {
+      const data = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        toxicExposure: { conditions: {} },
+      };
+
+      renderPage({ data });
+
+      const continueButton = document.querySelector(
+        '.usa-button-primary[type="submit"]',
+      );
+      expect(continueButton).to.exist;
+      expect(continueButton.textContent).to.contain('Continue');
+
+      const backButton = document.querySelector(
+        '.form-progress-buttons .usa-button-secondary',
+      );
+      expect(backButton).to.exist;
+      expect(backButton.textContent).to.contain('Back');
+    });
+
+    it('displays update button on review page', () => {
+      const data = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        toxicExposure: { conditions: {} },
+      };
+
+      renderPage({ data, onReviewPage: true });
+
+      expect(document.querySelector('va-button[text="Update page"]')).to.exist;
+      expect(
+        document.querySelector(
+          '.usa-button-primary[type="submit"][text="Continue"]',
+        ),
+      ).not.to.exist;
+      expect(
+        document.querySelector('.form-progress-buttons .usa-button-secondary'),
+      ).not.to.exist;
+    });
+  });
+
+  describe('Modal content', () => {
+    it('displays correct modal title and content', async () => {
+      const data = {
+        newDisabilities: [{ condition: 'Chronic Bronchitis' }],
+        toxicExposure: {
+          conditions: { none: true },
+          gulfWar1990: { bahrain: true },
+        },
+      };
+
+      const { user } = renderPage({ data, featureToggleEnabled: true });
+      const submitButton = document.querySelector('button[type="submit"]');
+
+      await user.click(submitButton);
+
+      await waitFor(() => {
+        const modal = document.querySelector('va-modal');
+        expect(modal).to.exist;
+
+        // Check modal title
+        expect(modal.innerHTML).to.contain(
+          'Remove condition related to toxic exposure?',
+        );
+
+        // Check modal content lists
+        expect(modal.innerHTML).to.contain(
+          'Gulf War service locations and dates (1990 and 2001)',
+        );
+        expect(modal.innerHTML).to.contain(
+          'Agent Orange exposure locations and dates',
+        );
+        expect(modal.innerHTML).to.contain(
+          'Other toxic exposure details and dates',
+        );
```

Comment:
So, these other toxic exposure locations and details are listed, even though only 1990 Gulf War corresponds to the data?